### ISSUE TITLE
Translation: r-gapminder table of contents

### ISSUE THREAD

**Issue by joelnitta** [swcarpentry-ja/i18n#111]
Translation: r-gapminder table of contents
-----

http://swcarpentry.github.io/r-novice-gapminder/

The table of contents is automatically generated from the front-matter of each episode. These all need to be translated. (Note that the comments such as `"# Please do not edit this file directly; it is auto generated.\n" ` and ` "# Instead, please edit 01-rstudio-intro.md in _episodes_rmd/\n" ` do not need to be translated)

目次はそれぞれのエピソードのフロントマターから自動的に作られます。全部翻訳する必要があります。（注：`"# Please do not edit this file directly; it is auto generated.\n" ` や ` "# Instead, please edit 01-rstudio-intro.md in _episodes_rmd/\n" `みたいなコメントは翻訳は要りません。）

* Please [see the README for more information about editing PO files](https://github.com/swcarpentry-ja/i18n/blob/ja/README_en.md#about-po-files)
* Please assign yourself (or add a comment) to the issue **before** you start translating, so other people know that you are translating.
* Once you have finished translating, send a pull request and reference this issue.

-----

* PO fileの編集については[READMEをご参照ください](https://github.com/swcarpentry-ja/i18n#about-po-files)。
* あなたが翻訳し始めたと分かるように、翻訳を**始める前**にこのイシューを自分に割り振って(assign)、あるいはコメントを書いて下さい。
* 翻訳が終わったら、プルリクエストを開いて、このイシューを参照・リンクして下さい。

https://github.com/swcarpentry-ja/i18n/blob/e8e5ac3fac015064959ef95db832756f93e02370/po/r-novice-gapminder.ja.po#L795-L856

https://github.com/swcarpentry-ja/i18n/blob/e8e5ac3fac015064959ef95db832756f93e02370/po/r-novice-gapminder.ja.po#L3262-L3299

https://github.com/swcarpentry-ja/i18n/blob/e8e5ac3fac015064959ef95db832756f93e02370/po/r-novice-gapminder.ja.po#L3781-L3816

https://github.com/swcarpentry-ja/i18n/blob/e8e5ac3fac015064959ef95db832756f93e02370/po/r-novice-gapminder.ja.po#L4392-L4435

https://github.com/swcarpentry-ja/i18n/blob/e8e5ac3fac015064959ef95db832756f93e02370/po/r-novice-gapminder.ja.po#L7078-L7133

https://github.com/swcarpentry-ja/i18n/blob/e8e5ac3fac015064959ef95db832756f93e02370/po/r-novice-gapminder.ja.po#L8354-L8397

https://github.com/swcarpentry-ja/i18n/blob/e8e5ac3fac015064959ef95db832756f93e02370/po/r-novice-gapminder.ja.po#L11061-L11098

https://github.com/swcarpentry-ja/i18n/blob/e8e5ac3fac015064959ef95db832756f93e02370/po/r-novice-gapminder.ja.po#L12271-L12312

https://github.com/swcarpentry-ja/i18n/blob/e8e5ac3fac015064959ef95db832756f93e02370/po/r-novice-gapminder.ja.po#L13269-L13300

https://github.com/swcarpentry-ja/i18n/blob/e8e5ac3fac015064959ef95db832756f93e02370/po/r-novice-gapminder.ja.po#L14176-L14223

https://github.com/swcarpentry-ja/i18n/blob/e8e5ac3fac015064959ef95db832756f93e02370/po/r-novice-gapminder.ja.po#L15468-L15501

https://github.com/swcarpentry-ja/i18n/blob/e8e5ac3fac015064959ef95db832756f93e02370/po/r-novice-gapminder.ja.po#L15877-L15908

https://github.com/swcarpentry-ja/i18n/blob/e8e5ac3fac015064959ef95db832756f93e02370/po/r-novice-gapminder.ja.po#L16793-L16836

https://github.com/swcarpentry-ja/i18n/blob/e8e5ac3fac015064959ef95db832756f93e02370/po/r-novice-gapminder.ja.po#L17962-L17997

https://github.com/swcarpentry-ja/i18n/blob/e8e5ac3fac015064959ef95db832756f93e02370/po/r-novice-gapminder.ja.po#L19078-L19123

https://github.com/swcarpentry-ja/i18n/blob/e8e5ac3fac015064959ef95db832756f93e02370/po/r-novice-gapminder.ja.po#L19911-L19954

**Comment by TomKellyGenetics** [comment]
Thanks for tracking down where these are the in PO file. This helps a lot!

**Comment by TomKellyGenetics** [comment]
Cross-posting from Slack in case others aren't following there. This is the largest remaining section along with #105 and #112 to complete a full version of the R course. This seems suitable to split up into tiny chunks and assign to many people "hackathon" style. Due to scheduling difficulties, these don't have to be done at the same time.

Rather than assigning yourself to the entire issue, how about we volunteer for each lessons one-at-a-time? Just comment here if you're doing one so we don't overlap. :)

**Comment by TomKellyGenetics** [comment]
Just to note here: @masamiy has started translating this.

**Comment by TomKellyGenetics** [comment]
I think all that needs to be translated here are the bullet points still in English on the web version: https://swcarpentry-ja.github.io/r-novice-gapminder/ja/_episodes/01-rstudio-intro/index.html

That is, we only need to change the bullet points (lines starting with `"- \"`)

For the headings, changes in `lang.md` will be applied across all lessons.

**Comment by joelnitta** [comment]
Yes; those are precisely the lines I've indicated above, in the original issue.

**Comment by TomKellyGenetics** [comment]
Agreed, just to clarify, these lines don't need to be translated except for the actual title. Not sure if the English terms are needed for the template.

https://github.com/swcarpentry-ja/i18n/blob/e8e5ac3fac015064959ef95db832756f93e02370/po/r-novice-gapminder.ja.po#L11067-L11070

https://github.com/swcarpentry-ja/i18n/blob/e8e5ac3fac015064959ef95db832756f93e02370/po/r-novice-gapminder.ja.po#L11073

This is all that needs to be change (and the bullet points).

```
"title: 制御フロー\n" 
 "teaching: 45\n" 
 "exercises: 20\n" 
 "questions:\n" 
```

**Comment by joelnitta** [comment]
resolved by PR #114
